client/src/main/java/org/evosuite/setup/DependencyAnalysis.java

```diff
@@ -104,10 +96,21 @@ private static void analyze(String className, List<String> classPath) throws Run
 		}
 
 		// TODO: Need to make sure that all classes in calltree are instrumented
-
 		logger.debug("Update call tree with calls to overridden methods");
 		CallGraphGenerator.update(callGraph, inheritanceTree);
 
+	}
+
+	private static void analyze(String className, List<String> classPath) throws RuntimeException,
```

Comment:
Suggestion: remove argument `classPath` since it's never used

Reply:
OK, done.